streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/WordCountTest.scala

```diff
@@ -87,8 +87,8 @@ class WordCountTest extends JUnitSuite with WordCountTestData {
     // generate word counts
     val wordCounts: KTable[String, Long] =
       textLines.flatMapValues(v => pattern.split(v.toLowerCase))
-        .groupBy((k, v) => v)
-        .count()
+        .groupBy((_, v) => v)
+        .count
```

Comment:
We should call `count()` here?

Reply:
Arf, missed this one. Thanks for picking it